frontend/packages/ceph-storage-plugin/src/components/independent-mode/utils.ts

```diff
@@ -56,3 +90,13 @@ export const getClusterHealth = (cluster: K8sResourceKind, loaded: boolean, erro
   if (!_.isEmpty(cluster)) return PhaseToState[phase];
   return HealthState.NOT_AVAILABLE;
 };
+
+export const prettifyJSON = (data: string) =>
+  _.isEmpty(data)
+    ? ''
+    : (() => {
+        const jsonData = JSON.parse(data);
```

Comment:
We might need to catch the error here in case JSON is not valid.

Reply:
Just noticed that you have checked the validity of JSON in `onUpload` function, so it might not be needed here.

Reply:
Yes, this comment is not applicable.